x-pack/platform/plugins/shared/alerting/server/routes/rule/apis/create/create_rule_route.ts

```diff
@@ -38,6 +39,7 @@ export const createRuleRoute = ({ router, licenseState, usageCounter }: RouteOpt
         access: 'public',
         summary: `Create a rule`,
         tags: ['oas-tag:alerting'],
+        oasFilePath: () => path.join(__dirname, 'create_rule.examples.yaml'),
```

Comment:
This is where we link the OAS in. Use of `__dirname` is sub-optimal with ECMAScript modules, but should be a straightforward find-and-replace with [import.meta.dirname](https://nodejs.org/api/esm.html#importmetadirname) if we ever get there.

Reply:
Could be cooler if we found a way to avoid this path wrangling. Like if it was just `oasFilePath: () => 'create_rule.examples.yaml'` or if we had a file path convention so that we could auto-link the OAS without code changes, but simplest seems best for now.